### PR TITLE
Feature/version definition

### PR DESCRIPTION
**Pull request by glow-mdsol**
@ykitamura-mdsol @isparks @ChazzMaTazz @jcarres-mdsol 
- fixed the version detection which failed when building the docs on readthedocs (it should automatically build https://requests-mauth.readthedocs.io)
- added more examples
- more clarifying documentation per the expected standards

### CODE REVIEW

File: docs/examples.rst [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+        return auth
+
+    def get_user_details(configuration, user_uuid):
+        """
+        Get the User details from the iMedidata API
+        :param dict configuration: Configuration Set
+        :param str user_uuid: UUID for user
+        """
+        mauth_signer = generate_signer(**configuration)
+        base_url = "https://innovate.imedidata.com"
```
Comment (ssteeg-mdsol) [inline docs/examples.rst]: Any particular reason for using innovate over the production URL?

File: docs/examples.rst [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+        else:
+            auth = MAuth(app_uuid, private_key_string)
+        return auth
+
+    def get_countries(configuration):
+        """
+        Get the list of countries from the API GW
+        :param dict configuration: a configuration dictionary
+        """
+        full_url = "https://apigw.imedidata.com/v1/countries"
```
Comment (ssteeg-mdsol) [inline docs/examples.rst]: We recently decided to change the domain for the API Gateway. The new URL would be `https://api.mdsol.com/v1/countries`

File: docs/examples.rst (hunk 1 above)
Comment (glow-mdsol) [reply docs/examples.rst]: No particular reason - I had credentials that worked I guess.  I've updated.

File: docs/examples.rst (hunk 2 above)
Comment (glow-mdsol) [reply docs/examples.rst]: Ok, I copied the example from a ruby sample that someone shared with me.

**Pull request merged**